incubator/hnc/pkg/validators/hierarchy.go

```diff
@@ -103,6 +105,12 @@ func (v *Hierarchy) handle(ctx context.Context, log logr.Logger, req *request) a
 		return allow("HNC SA")
 	}
 
+	// Do pre-check namespaces in the request configuration, make sure
```

Comment:
Finish this comment? Maybe something like:

`// Verify the HC is legal in isolation (i.e., before checking the rest of the forest)`

Reply:
ok